src/pages/profileFill/bussInfo.js

```diff
@@ -0,0 +1,590 @@
+import React, { useState } from "react";
+import styles from "./bussInfo.module.css";
+import { useRef } from "react";
+import { Link } from "react-router-dom";
+const BussInfo = () => {
+  const [count, setCount] = useState(80);
```

Comment:
I dont think we need this much states to handle here in this page .Please refactor your code for state mannagement in all pages. This is not an optimised code.